stdlib/src/math.rs

```diff
@@ -612,19 +629,19 @@ mod math {
 
     #[pyfunction]
     fn fsum(seq: ArgIterable<ArgIntoFloat>, vm: &VirtualMachine) -> PyResult<f64> {
-        let mut partials = vec![];
+        let mut partials = Vec::with_capacity(32);
         let mut special_sum = 0.0;
         let mut inf_sum = 0.0;
 
         for obj in seq.iter(vm)? {
             let mut x = *obj?;
 
             let xsave = x;
-            let mut j = 0;
```

Comment:
Do you have a reason to switch j and i?

Reply:
This is to match the logic in the CPython implementation closer

https://github.com/python/cpython/blob/94cd2e0ddeff83dee3254ca356d9e4396927d075/Modules/mathmodule.c#L1398-L1409

Where `j` is the actual iterator of the whole list and `i` is a "write index".